tests/parsing.tests.ts

```diff
@@ -89,44 +122,68 @@ describe("request parsing tests", () => {
             resource: "from_path_test",
             method: "GET",
             pathParameters: {
-                test: "test_path_param",
+                string: "string",
+                number: "5",
+                boolean: "true",
             },
         })
 
         const response = await router.route(event, context)
 
         expect(response.statusCode).to.equal(200)
-        expect(response.body).to.equal("test_path_param")
+        expect(response.body).to.equal(
+            JSON.stringify({
+                string: "string",
+                number: 5,
+                boolean: true,
+            })
+        )
     })
 
     it("extracts query string parameter from request", async () => {
         const event = createAPIGatewayEvent({
             resource: "from_query_test",
             method: "GET",
             queryStringParameters: {
-                test: "test_query_param",
+                string: "string",
+                number: "5",
```

Comment:
what happens if this isn't actually a number? Controller gets `NaN`??

Wonder if we should throw a `RequestError`?

Reply:
Hmm, yeah might be better to throw a request error